### PR TITLE
test Julia 1.3, FreeBSD, add badge

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
Maybe we shouldn't force GR in the tests; it doesn't seem to install on FreeBSD.

**Comment by dpo** [comment]
Is Plots.jl unusable on FreeBSD? It insists on building GR, which always fails.

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+task:
+  name: FreeBSD
+  env:
+    matrix:
+      - JULIA_VERSION: 1.0
+      - JULIA_VERSION: 1.1
+      - JULIA_VERSION: 1.2
+      - JULIA_VERSION: 1.3
+      - JULIA_VERSION: nightly
+  allow_failures: JULIA_VERSION == 'nightly'
```
Comment (abelsiqueira) [inline .cirrus.yml]: I think this needs to be `$JULIA_VERSION` ([ref](https://github.com/cirruslabs/cirrus-ci-docs/commit/7d389ea688111ac2cc88aea12a15e7ae1756794e))

Comment (dpo) [reply .cirrus.yml]: Let's try. There was no complaint about the syntax.

**Comment by dpo** [comment]
Wanted to squash but clicked too fast. Now the commit message sucks. Oh well.

**Pull request merged**